cli/src/config/templates.toml

```diff
@@ -156,12 +140,26 @@ commit_summary_separator = 'label("separator", " | ")'
   time_range.start().ago() ++ label("time", ", lasted ") ++ time_range.duration()'''
 'format_timestamp(timestamp)' = 'timestamp.local().format("%Y-%m-%d %H:%M:%S")'
 
+format_op_log_operation = '''
```

Comment:
nit: since it's called `format_()`, it seems natural to take an Operation object. (i.e. `format_op_log_operation(op) = .. op.id().short() ..`)

Reply:
Huh, right, we can just pass `self` to that, can't we. Did that, it works :+1: 

Although there still will be `self` available, but yeah this makes clearer and more inline with other formats too